cmd/provisioner-localpv/app/start.go

```diff
@@ -55,6 +61,39 @@ func StartProvisioner() (*cobra.Command, error) {
 	return cmd, nil
 }
 
+// This function performs the preupgrade related tasks for 1.0 to 1.1
```

Comment:
@vishnuitta -- can we move this function into a seperate file called, backward-compatability.go